complex objects/ios/ComplexRust/Store/Logins/Login.swift

```diff
@@ -4,7 +4,7 @@
 
 import Foundation
 
-enum LoginStatus: Int {
+enum LoginStatus: UInt32 {
```

Comment:
I think if you make the enum `repr(C)`, this will be `UInt`, I think. Or perhaps not.